README.md

```diff
@@ -3,3 +3,33 @@
 ## Overview
 
 The DropAI suite comprises modules that empower Drupal websites with AI capabilities, facilitating the transformation of Drupal's data model into a Language Model (LLM). It facilitates the creation of embeddings from Drupal content and metadata, enabling efficient content management and transformation into a vector database.
+
+## PDF Reader
+
+This module provides integration with PDF files, its implementation allows managing different libraries for converting a PDF to plain text, some of them:
+
+### PDF Paser:
+
+The smalot/pdfparser is a standalone PHP package that provides various tools to extract data from PDF files.
+
+This library is under active maintenance. There is no active development by the author of this library (at the moment), but we welcome any pull request adding/extending functionality!
+
+**Installation:**
+
+Add the library to composer.
+
+`lando composer require smalot/pdfparser`
+
+Autoload the library to the project, you need to add these lines to the composer.json file:
+
+```
```

Comment:
You do not need to manually autoload this path. This project is unique because we are organizing it like a contrib modules and eventually it will be moved to Drupal.org. If you add the library to your composer.json file, then composer will automatically load it when our dropai module is installed.